cid/helpers/cur_proxy.py

```diff
@@ -0,0 +1,452 @@
+    'product_network_performance': "product['network_performance']",
+    'product_normalization_size_factor': "product['normalization_size_factor']",
+    'product_operating_system': "product['operating_system']",
+    'product_physical_processor': "product['physical_processor']",
+    'product_platopricingtype': "product['platopricingtype']",
+    'product_platovolumetype': "product['platovolumetype']",
+    'product_pre_installed_sw': "product['pre_installed_sw']",
+    'product_processor_architecture': "product['processor_architecture']",
+    'product_processor_features': "product['processor_features']",
+    'product_queue_type': "product['queue_type']",
+    'product_region': "product['region']",
+    'product_request_type': "product['request_type']",
+    'product_servicename': "product['servicename']",
+    'product_storage': "product['storage']",
+    'product_storage_class': "product['storage_class']",
+    'product_storage_media': "product['storage_media']",
+    'product_tenancy': "product['tenancy']",
+    'product_transfer_type': "product['transfer_type']",
+    'product_vcpu': "product['vcpu']",
+    'product_version': "product['version']",
+    'product_volume_api_name': "product['volume_api_name']",
+    'product_volume_type': "product['volume_type']",
+    'product_vpcnetworkingsupport': "product['vpcnetworkingsupport']",
+    'product_edition': "product['edition']",
+    'product_gpu_memory': "product['gpu_memory']",
+    'product_pack_size': "product['pack_size']",
+    'product_q_present': "product['q_present']",
+    'product_subscription_type': "product['subscription_type']",
+    'product_usage_group': "product['usage_group']",
+    'product_cache_engine': "product['cache_engine']",
+    'product_invocation': "product['invocation']",
+    'product_memory_gib': "product['memory_gib']",
+    'product_time_window': "product['time_window']",
+    'product_finding_group': "product['finding_group']",
+    'product_finding_source': "product['finding_source']",
+    'product_finding_storage': "product['finding_storage']",
+    'product_standard_group': "product['standard_group']",
+    'product_standard_storage': "product['standard_storage']",
+    'product_product_type': "product['product_type']",
+    'product_pricingplan': "product['pricingplan']",
+    'product_provider': "product['provider']",
+    'product_subservice': "product['subservice']",
+    'product_type': "product['type']",
+    'product_tickettype': 'n/a',
+    'product_memorytype': 'n/a',
+    'product_platousagetype': 'n/a',
+    'product_with_active_users': 'n/a',
+    'product_abd_instance_class': 'n/a',
+    'product_size_flex': 'n/a',
+    'product_engine_major_version': 'n/a',
+    'product_extended_support_pricing_year': 'n/a',
+    "concat('name', bill_payer_account_id)": 'bill_payer_account_name',
+    "concat('name', line_item_usage_account_id)": 'line_item_usage_account_name',
+}
+
+default_columns = {
```

Comment:
What is purpose of default columns compared to mapping above?

Reply:
Mapping above contains all 
Default - the most used.

Reply:
And why we need this from logic perspective? Any specific benefits it brings?

Reply:
With a reasonable set of default dashboards authors do not need to care about describing cur fields. 

cid-cmd export cannot do that today, so it stays a manual operation for authors.